WcaOnRails/app/models/competition.rb

```diff
@@ -1642,7 +1642,7 @@ def persons_wcif(authorized: false)
                                 .includes(includes_associations)
                                 .to_enum
                                 .with_index(1)
-                                .select { |r, registrant_id| authorized || r.accepted? }
+                                .select { |r, registrant_id| authorized || r.accepted? || !r.is_competing? }
```

Comment:
Might be worth adding a comment here why this extra `is_competing` check is necessary, for people who don't know that we (currently) don't mark non-competing staff as "accepted" in the UI.

Reply:
Done.

Reply:
Now you have an extra method below but you don't use it up here...
What about `.select { |r, registrant_id| authorized || r.wcif_status == "accepted" }`?

Reply:
Sure, done.